napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -903,14 +903,22 @@ def get_interfaces_ip(self):
             #   2001:cc11:22bb:0:2ec2:60ff:fe4f:feb2/64 [VALID]
             # IPv6 subnet:  2001::/24
             # IPv6 link-local address: fe80::2ec2:60ff:fe4f:feb2 (default) [VALID]
+            # IPv6 address: fe80::a293:51ff:fe5f:5ce9 [VALID]
             if "Interface status" in line:
                 interface = line.split(",")[0]
                 continue
             if "VALID" in line:
                 line = line.strip()
                 if "link-local address" in line:
+                    # match the following format:
+                    # IPv6 link-local address: fe80::2ec2:60ff:fe4f:feb2 (default) [VALID]
                     ip_address = line.split()[3]
                     prefix_len = "64"
+                elif "IPv6 address" in line:
+                    # match the following format:
+                    # IPv6 address: fe80::a293:51ff:fe5f:5ce9 [VALID]
+                    ip_address = line.split()[2]
+                    prefix_len = "64"
```

Comment:
Are addresses of this form always link-local? Should we put a check in where if `ip_address` contains a `/` then raise an exception. I am a bit worried that we might have a case where we incorrectly assume a /64 prefix when it isn't.

Maybe a better solution would just be to move the case where we search for '/' up (move it up from the else clause and change the logic a bit). Structurally something like:

 ```
if "link-local address" in line:
    ip_address = line.split()[3]
    prefix_len = "64"
elif "/" in line:
    ip_address, prefix_len = line.split()[0].split("/")
elif "IPv6 address" in line:
    ip_address = line.split()[2]
    prefix_len = "64"
else:
    raise ValueError("We don't know what just happened here")
```